Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
+	release-victoria-metrics-linux-amd64 \
+	release-victoria-metrics-darwin-amd64 \
 	release-victoria-metrics-arm \
+	release-victoria-metrics-linux-arm \
 	release-victoria-metrics-arm64 \
-	release-victoria-metrics-darwin-amd64 \
+	release-victoria-metrics-linux-arm64 \
 	release-victoria-metrics-darwin-arm64
 
 release-victoria-metrics-amd64:
```

Comment:
This target should be removed in favor of `release-victoria-metrics-linux-amd64`